src/applications/representative-appoint/components/ProfileNotUpdatedNote.jsx

```diff
@@ -31,7 +32,7 @@ function ProfileNotUpdatedNote(props) {
           {includeLink && (
             <>
               <va-link
-                href="https://va.gov"
+                href={`https://${environment.BASE_URL}/change-address`}
```

Comment:
This link doesn't work locally. Also, the `(opens in new tab)` text is repeated:
![image](https://github.com/user-attachments/assets/7432ce20-955a-4287-9144-e278fa1fe781)

Reply:
The link wouldnt work locally when only building `representative-appoint` locally since it's directing away from our app. However, the leading `https://` should be removed. Looks like it's now forming `localhost:3001/change-address` in a new tab, which is the expected outcome. 

Removed the duplicate text.